gribscan/tools.py

```diff
@@ -7,136 +6,109 @@
 from pathlib import Path
 import multiprocessing as mp
 
+import click
+
 import gribscan
 from .magician import MAGICIANS
 
 
-def create_index():
-    parser = argparse.ArgumentParser()
-    parser.add_argument("sources", metavar="GRIB", help="source gribfile(s)", nargs="+")
-    parser.add_argument(
-        "-v",
-        "--verbose",
-        help="increase the logging level",
-        action="store_true",
-    )
-    parser.add_argument(
-        "-o",
-        "--outdir",
-        help="output directory to write index files",
-        type=str,
-        default=None,
-        nargs="?",
-    )
-    parser.add_argument(
-        "-f",
-        "--force",
-        help="overwrite existing index files",
-        action="store_true",
-    )
-    parser.add_argument(
-        "-n",
-        "--nprocs",
-        help="number of parallel processes",
-        type=int,
-        default=1,
-        nargs="?",
-    )
-    args = parser.parse_args()
-
+@click.group()
+@click.option("-v", "--verbose", is_flag=True, help="Increase the logging level.")
```

Comment:
This change will cause a change in behaviour, as one couldn't do `gribscan-build -v` anymore. Probably that's ok, as one likely doesn't use `-v` in any automated code, but still.